pkg/ui/src/components/core/colors.styl

```diff
@@ -47,13 +47,17 @@ $colors--primary-green-3 = #37a806
 $colors--primary-green-4 = #237300
 $colors--primary-green-5 = #123d00
 
+$colors--primary-purple-2 = #CBBCFC
+$colors--primary-purple-3 = #6933FF
+
+$colors--info-4 = #0037a5
+
 // Functional colors
 $colors--functional-purple-1 = #cec0f2
```

Comment:
I see the "primary" color palette (purple) defined in Figma. Where do the "functional-purple" colors come from?

Reply:
those are some old vars. from previous design versions I guess..